index.html

```diff
@@ -2,12 +2,122 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-
-    <link href="styles.css" rel="stylesheet" />
-
-    <title>Rocket Academy</title>
+    <link href="portfolio.css" rel="stylesheet" />
+    <title>My Portfolio</title>
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <!-- Navigation Bar -->
+    <div class="navbar">
+      <a href="index.html">Home</a>
+      <a href="#">About Me</a>
+      <a href="#">Portfolio</a>
+      <a href="#">Contact</a>
+    </div>
+
+    <!-- Header -->
+    <div id="main-border">
+      <div class="main-header">
+        <div class="sub-header">
+          <div id="hello">HELLO I AM</div>
+          <div id="my-name">POH WEN XIN</div>
+          <div id="my-intro">
+            I aspire to design and build beautiful websites for businesses
+            around the globe. If you need a modern and powerful website, simply
+            send me an
+            <span style="color: hsl(40, 95%, 54%)">email</span> or visit my
+            <span style="color: hsl(40, 95%, 54%)">Linkedin profile</span>.
+            <div id="discover-btn">
+<a href="https://www.linkedin.com/">DISCOVER NOW</a></div>
+          </div>
+        </div>
+        <div class="sub-header">
+          <img src="Profile Pic.jpg" width="80%;" />
+        </div>
+      </div>
+    </div>
+
+    <!-- The flexible grid (content) -->
+    <div class="main-container">
+      <h1>MY PROJECTS.</h1>
+
+      <!-- Subcontainer 1 -->
+     <div class="flip-card">
+     <div class="flip-card-inner">
+     <div class="flip-card-front">
+       <img src="Just Dough It.jpeg" alt="Avatar" style="width:300px;height:200px;">
+     </div>
+      <div class="flip-card-back">
+        <h1>Recipe Site</h1></a>
+      <p><i>A website design created to store recipes.</i></p>
+      <p>October 2024</p>
+       <p><a href="https://wenxin-p.github.io/recipe-site-bootcamp/index.html"><u>Visit Site</u></p></a>
+    </div>
+  </div>
+</div>
+
+  <!-- Subcontainer 2 -->
+     <div class="flip-card">
+     <div class="flip-card-inner">
+     <div class="flip-card-front">
+       <img src="Coming Soon Logo.jpg" alt="Coming Soon" style="width:300px;height:200px;">
+     </div>
+      <div class="flip-card-back">
+      <h1>Title of Project</h1>
+      <p>Description of Project</p>
+      <p>Date of Completion</p>
+    </div>
+  </div>
+</div>
+
+  <!-- Subcontainer 3 -->
+     <div class="flip-card">
+     <div class="flip-card-inner">
+     <div class="flip-card-front">
+        <img src="Coming Soon Logo.jpg" alt="Coming Soon" style="width:300px;height:200px;">
```

Comment:
let's avoid inline styles like here and use css. Alternatively you could just use the `width` and `heigth` attribute on an `img` tag